scalability_and_performance/ztp-factory-install-clusters.adoc

```diff
@@ -0,0 +1,127 @@
+
+* Deploying {product-title} Data Foundation, see link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_data_foundation_using_bare_metal_infrastructure/index[Deploying OpenShift Data Foundation using bare metal infrastructure].
+
+* Red Hat Advanced Cluster Management, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4[Product Documentation for Red Hat Advanced Cluster Management for Kubernetes 2.4
+].
+
+include::modules/ztp-intro-factory-install.adoc[leveloffset=+1]
+
+include::modules/ztp-install-factory-pipeline-overview.adoc[leveloffset=+1]
+
+include::modules/ztp-hub-factory-pipeline.adoc[leveloffset=+2]
+
+.Additional resources
+
+For more information about:
+
+* `opm` and `oc`, see xref:../cli_reference/index.adoc#cli-tools-overview[CLI tools overview].
+
+* Installing Red Hat Advanced Cluster Management (RHACM), see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/install/installing[Install and uninstall Red Hat Advanced Cluster Management].
+* Enabling the Assisted Installer on {product-title}, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#enabling-assisted-installer-service-on-bare-metal_ztp-deploying-disconnected[Enabling assisted installer service on bare metal].
+
+include::modules/ztp-spoke-factory-pipeline.adoc[leveloffset=+2]
+
+include::modules/ztp-pre-hub-install-factory-pipeline-checks.adoc[leveloffset=+1]
+
+.Additional resources
+
+For more information about:
+
+* The NMState Operator see, xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc[About the Kubernetes NMState Operator].
+
+include::modules/ztp-install-openshift-pipelines.adoc[leveloffset=+1]
+
+include::modules/ztp-install-factory-pipeline.adoc[leveloffset=+1]
+
+.Additional resources
+
+For more information about the supported flags and arguments see, xref:../scalability_and_performance/ztp-factory-install-clusters.adoc#ztp-factory-install-pipelines-flags-arguments_ztp-factory-install[ZTP factory install pipelines flags and arguments].
+
+include::modules/ztp-check-install-hub-factory-pipeline.adoc[leveloffset=+2]
+
+include::modules/ztp-post-install-hub-factory-pipeline-checks.adoc[leveloffset=+2]
+
+include::modules/ztp-install-factory-spoke-pipeline.adoc[leveloffset=+1]
+
+include::modules/ztp-check-install-spoke-factory-pipeline.adoc[leveloffset=+2]
+
+include::modules/ztp-post-install-spoke-factory-pipeline-checks.adoc[leveloffset=+2]
+
+include::modules/ztp-troubleshooting-factory-pipelines.adoc[leveloffset=+1]
+
+include::modules/ztp-common-expected-errors.adoc[leveloffset=+1]
+
+include::modules/ztp-configuring-spoke-at-remote-site.adoc[leveloffset=+1]
+
+include::modules/ztpfw-pipelines-flags-arguments.adoc[leveloffset=+1]
+
+[id="{context}-additional-resources"]
```

Comment:
Repo guidelines specify a unique ID in a different format than is used here. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies